web/app/webpack.config.js

```diff
@@ -76,6 +76,7 @@ module.exports = {
     }),
     // compile the bundle with hashed filename into index_bundle.js
     new HtmlWebpackPlugin({
+      cache: false,
```

Comment:
The dashboard finds all sorts of ways to get cached!

Reply:
This one was a weird one. `HtmlWebpackPlugin` just released a new version a month or so ago. In it they *massively* reduced the time spent. Some of that was by moving over to a new hooks API and some of it was apparently related to doing some intelligent caching. For our production builds, that's awesome. For development, the bundle gets deleted because it isn't changing when you update a single file. Interactions between libraries continue to teach me new and interesting things.